indico/modules/events/models/persons.py

```diff
@@ -208,6 +161,54 @@ def person_id(cls):
         )
 
     @declared_attr
+    def _first_name(cls):
+        return db.Column(
+            'first_name',
+            db.String,
+            nullable=True,
```

Comment:
I just went throughout our models quickly, we don't keep the  `,` at the last argument.